skipper.env

```diff
@@ -102,3 +102,5 @@ INSTALLATION_TIMEOUT
 JUNIT_REPORT_DIR
 CI
 NETWORK_TYPE
+OS_IMAGES
+RELEASE_IMAGES
```

Comment:
Please also add these to the readme: https://github.com/openshift/assisted-test-infra#installation-parameters